spec/namespaces/_core.yaml

```diff
@@ -3334,6 +3334,7 @@ components:
         Comma-separated list of scroll IDs to clear.
         To clear all scroll IDs, use `_all`.
       required: true
+      deprecated: true
```

Comment:
Does this need to be specified here when it's already specified at the path level? https://github.com/opensearch-project/opensearch-api-specification/blob/84a31b0c4e38ad0f035e887fabbd0ab6701c39ce/spec/namespaces/_core.yaml#L880-L883

Reply:
Here is a similar endpoint `scroll::path.scroll_id`. Deprecation is present in both places. 

https://github.com/opensearch-project/opensearch-api-specification/blob/84a31b0c4e38ad0f035e887fabbd0ab6701c39ce/spec/namespaces/_core.yaml#L5128